src/backEnd.js

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable import/no-mutable-exports */
 
-let tasks = [
-  { description: 'Laundary', completed: true, index: 1 },
-  { description: 'Car wash', completed: false, index: 2 },
-  { description: 'Swimming', completed: false, index: 3 },
-];
+let tasks = [];
+
+const loadTaskList = () => {
+  let loadTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (loadTasks == null) {
+    loadTasks = [];
+  }
```

Comment:
- [ ] After deleting a todo item and refreshing the page the item still returns to the page. Kindly fix it.